vault/logical_passthrough.go

```diff
@@ -147,6 +149,65 @@ func (b *PassthroughBackend) handleRead(
 	return resp, nil
 }
 
+func (b *PassthroughBackend) handleMultiple(req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+	// Prepare paths
+	pathsRaw := req.Get("paths")
+	if pathsRaw == nil {
+		return nil, fmt.Errorf("no paths were specified")
+	}
+
+	// If it's not a slice, return error
+	if reflect.TypeOf(pathsRaw).Kind() != reflect.Slice {
+		return nil, fmt.Errorf("paths must be specified as array")
+	}
+
+	pathsRawSlice := pathsRaw.([]interface{})
+	paths := make([]string, len(pathsRawSlice))
+	for i, v := range pathsRawSlice {
+		paths[i] = fmt.Sprint(v)
+	}
+
+	// Get all secrets
+	secrets := make(map[string]interface{})
+	for _, path := range paths {
+		out, err := req.Storage.Get(path)
+
+		if err != nil || out == nil {
+			continue
```

Comment:
Maybe it's possible to collect all errors to a slice and show this slice in the response? So, in this case, if client want to handle these errors somehow, it's possible to do it.